### ISSUE TITLE
Missing data in prediction

### ISSUE THREAD

**Issue by victorxie996** [hennande/Temporal_Relational_Stock_Ranking#15]
Hi!

Appreciate for sharing such greate work!

My concern regarding to the pratical usability of this whole pipeline is: if you select a stock universe i.e., stocks in the CSI300 index, or just simply select n stocks (doesn't really matter), to train the model, then how is the model gonna functioning well (do the right prediction) if some stocks quit the market, or stopped trading for a short period? From my understanding the vertices and edges cannot be missing, and imputing values to nan would lead to inaccurate prediction?

**Comment by QuantHao** [comment]
Agree, this is why I am so far still not fully trust GNN could be useful.
